hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerStateMachineStream.java

```diff
@@ -113,20 +112,18 @@ public void setup() throws Exception {
     raftClientConfig.setRpcWatchRequestTimeout(Duration.ofSeconds(10));
     conf.setFromObject(raftClientConfig);
 
-    ClientConfigForTesting.newBuilder(StorageUnit.BYTES)
-        .setDataStreamMinPacketSize(1024)
-        .setBlockSize(BLOCK_SIZE)
-        .setChunkSize(CHUNK_SIZE)
-        .setStreamBufferFlushSize(FLUSH_SIZE)
-        .setStreamBufferMaxSize(MAX_FLUSH_SIZE)
-        .applyTo(conf);
-
     conf.setLong(OzoneConfigKeys.DFS_RATIS_SNAPSHOT_THRESHOLD_KEY, 1);
     conf.setQuietMode(false);
     cluster =
         MiniOzoneCluster.newBuilder(conf)
             .setNumDatanodes(3)
             .setHbInterval(200)
+            .setDataStreamMinPacketSize(1024)
```

Comment:
The changes made under this test class seem unintended, could you please address these?